src/main/java/uk/gov/hmcts/darts/transcriptions/validator/WorkflowValidator.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.darts.transcriptions.validator;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum;
+import uk.gov.hmcts.darts.transcriptions.enums.TranscriptionTypeEnum;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.APPROVED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.AWAITING_AUTHORISATION;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.CLOSED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.COMPLETE;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.REJECTED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.REQUESTED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.WITH_TRANSCRIBER;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionTypeEnum.OTHER;
+
+@Component
+@Slf4j
+public class WorkflowValidator {
+
+    // Create a map defining the transition rules, from any given current state to a list of allowable target states
+    private final Map<TranscriptionStatusEnum, Set<TranscriptionStatusEnum>> manualWorkflowTransitionRules = new HashMap<>();
+
+    private final Map<TranscriptionStatusEnum, Set<TranscriptionStatusEnum>> automaticWorkflowTransitionRules = new HashMap<>();
+
+    public WorkflowValidator() {
+        manualWorkflowTransitionRules.put(REQUESTED, Set.of(REQUESTED, AWAITING_AUTHORISATION, CLOSED));
+        manualWorkflowTransitionRules.put(AWAITING_AUTHORISATION, Set.of(APPROVED, REJECTED, CLOSED));
+        manualWorkflowTransitionRules.put(APPROVED, Set.of(WITH_TRANSCRIBER, CLOSED));
+        manualWorkflowTransitionRules.put(REJECTED, Collections.emptySet());
+        manualWorkflowTransitionRules.put(WITH_TRANSCRIBER, Set.of(COMPLETE, CLOSED));
+        manualWorkflowTransitionRules.put(COMPLETE, Collections.emptySet());
+        manualWorkflowTransitionRules.put(CLOSED, Collections.emptySet());
+
+        automaticWorkflowTransitionRules.put(REQUESTED, Set.of(REQUESTED, APPROVED, CLOSED));
```

Comment:
Same as manual - don't think we should have REQUESTED again, only APPROVED, CLOSED?